### DISCUSSION TITLE
help!help!help!!!When dpgen reproduces the example of CH4, the POTCAR in 02.fp is empty, causing record.dpgen to stop at step 07.

### DISCUSSION THREAD

**Discussion by 16611808637** [deepmodeling/dpgen#1146] [Q&A]
My dpgen is 0.11.0, because the POTCAR file in 02.fp is empty, so the operation failed, but I manually merged POTCAR_H and POTCAR_C together, using mpirun -np 16 vasp can run, I don't know what went wrong , 
![dpgen](https://user-images.githubusercontent.com/126969768/223134167-ce63714a-cad1-4949-b3f9-9ec8d619bb98.png)
I implore the boss to give me some guidance.

**Comment by 16611808637** [comment]
All generated files
[generated files and configuration files.zip](https://github.com/deepmodeling/dpgen/files/10904504/generated.files.and.configuration.files.zip)
 and configuration files are as follows

**Reply by luowh35** [reply]
try the straight  fp command "vasp_std" rather than "mpirun -np 16 vasp_std"

**Reply by 16611808637** [reply]
I'm sorry, but I don't know how to operate the method you mentioned. Could you elaborate?

**Reply by luowh35** [reply]
the fp command : "mpirun -np 16 /mnt/d/Software/vasp.5.4.4/bin/vasp_std" in machine.json , turn into "vasp_std"

**Reply by chb-deep** [reply]
I also encountered the problem in task07, how exactly did you solve it
